packages/craco/lib/cra.js

```diff
@@ -36,6 +38,26 @@ function overrideModule(modulePath, newModule) {
     log(`Overrided require cache for module: ${modulePath}`);
 }
 
+function resolvePackageJson(cracoConfig) {
+    return require.resolve(path.join(cracoConfig.reactScriptsVersion, "package.json"), { paths: [projectRoot] });
+}
+
+function getReactScriptVersion(cracoConfig) {
```

Comment:
Since this function doesn't seem to be used anywhere, the code could be moved directly into `getReactScriptVersionInformation` and you could rename `getReactScriptVersionInformation` to `getReactScriptVersion`